src/assets/css/style.css

```diff
@@ -352,9 +352,9 @@ footer #version {
   border-top-left-radius: 0;
 }
 
-.btn-group .btn+.btn, 
-.btn-group .btn+.btn-group, 
-.btn-group .btn-group+.btn, 
+.btn-group .btn+.btn,
```

Comment:
Hmm, I'm not sure why these few lines have different line endings than the rest. I think my IDE just auto-trimmed them. Is that OK?